docs/reference/snapshot-restore/snapshot-vers-compat.asciidoc

```diff
@@ -0,0 +1,10 @@
+
+[cols="^,^,^,^,^"]
+|====
+| 4+^h| Cluster version
+h| Snapshot version         | 5.0–5.6    | 6.0–6.8    | 7.0–{prev-major-last} | 8.0–{minor-version}
+| 5.0–5.6                   | {yes-icon} | {yes-icon} | {no-icon}             | {no-icon}
+| 6.0–6.8                   | {no-icon}  | {yes-icon} | {yes-icon}            | {no-icon}
```

Comment:
I think we need a note here that restoring a 6.8 snapshot on a 7.0 cluster is not possible, just like we do not support that [upgrade path](https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-upgrade.html#setup-upgrade). I am not really 100% sure this is true, so let us give a bit of time to hear from others on this. My understanding is that 6.8 is only compatible with 7.2+, both full cluster and rolling upgrade, and that the same applies to restoring from snapshot.

Reply:
Thanks @henningandersen. I did some local testing and was able to restore a snapshot containing a small 6.8 index to a 7.0 cluster. It seems like I could index, search, and configure this index normally. That said, the setup may not be supported.

Let me know if there's a specific person or group I can reach out to to get a definitive answer.

Reply:
Based on the offline discussion, I pushed 30500e84d0f939eb72c26aa121cf31a836eb4c44 to update the tables to indicate you can't restore a 6.8 snapshot or index to a 7.0 or 7.1 cluster.